docs/steamdeck-and-linux/installing-on-steamdeck-and-linux.md

```diff
@@ -6,9 +6,9 @@
 EA App currently displays a blank screen when using NorthstarProton, however Northstar will still launch assuming you have logged in to EA App at least once.
 {% endhint %}
 
-> **Check your GLIBC version.** NorthstarProton currently only supports version 2.33 and higher. Verify your installed version with `ldd --version`. **Ubuntu 20.04 LTS**, **Debian 11**, and **Void Linux** are known to have outdated GLIBC packages, meaning you will have to experiment with finding a Proton version that works for you, or use Lutris. It is therefore strongly recommended to use an up-to-date Linux distro to be able to make use of NorthstarProton. This check does not need to be completed on Steam Deck or Steam OS.
+> **Northstar does not work on Ubuntu and Debian distros.** The workaround to this is using Distrobox, which will be explained further down, or using a known supported distro like Fedora or Arch. You will still need to follow the instructions below to install Northstar with the Steam (Native Package) path.
```

Comment:
Actually, we should also mention that we do wanna look into this and maybe link to https://github.com/R2Northstar/Northstar/issues/469 which I just created now.

That way we could maybe even attract someone to look into this cause tbh we should have Northstar working on Ubuntu and Debian ootb (using NorthstarProton). They make up a big enough chunk of userbase that it's worth the effort of investigating.

Reply:
latest commit resolves